.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,7 @@ Please explain the changes you made here.
 
 - [ ] Tests
 - [ ] Documentation
-- [ ] If the bundle manifests have been updated, make sure to review the [`rhdh-operator.csv.yaml`](../.rhdh/bundle/manifests/rhdh-operator.csv.yaml) file accordingly
+- [ ] If the bundle manifests have been updated, make sure to review the [`rhdh-operator.clusterserviceversion.yaml`](../.rhdh/bundle/manifests/rhdh-operator.clusterserviceversion.yaml) file accordingly
```

Comment:
unless you're also renaming the file in the repo, this change doesn't apply in 1.3. IF we onboard 1.3.z to Konflux, then it would. But I'm not ready for that yet. See https://issues.redhat.com/browse/RHIDP-4188